libs/ui/src/themes/make_theme.ts

```diff
@@ -82,18 +81,6 @@ const colorThemes: Record<ColorMode, ColorTheme> = {
     foreground: Color.GRAY_LIGHT,
     foregroundDisabled: Color.GRAY_LIGHT,
   },
-
-  legacy: {
```

Comment:
could you also update the storybook preview setup code in `.storybook/preview.tsx`? makes me realise we need to add a storybook build step to the CI process to make sure it stays in sync

Reply:
I tested `storybook` on this branch which built just fine, but breaks when you try to select the still extant Legacy color or size options. That being the case, I'm not sure how we could catch this in an automated way.

Reply:
I think the problem is just that the `.storybook` dir isn't being type-checked. I added it to the tsconfig and that caught the problem.

Reply:
6bb3e7ea8